packages/web3-core-subscriptions/src/subscriptions/eth/LogSubscription.js

```diff
@@ -66,6 +66,7 @@ export default class LogSubscription extends AbstractSubscription {
                     });
 
                     delete this.options.fromBlock;
+                    delete this.options.toBlock;
```

Comment:
Thanks for opening this PR! Could you extend the test case for the deletion of the ``toBlock`` property?

Reply:
sorry only saw the message now, all stuff on tests are done right?